.github/workflows/windows.yml

```diff
@@ -179,6 +179,8 @@ jobs:
           Remove-Item extra\groonga\test -Recurse -Force
           cd extra\groonga\vendor\mruby-source
           Invoke-WebRequest `
+            -MaximumRetryCount 3 `
+            -RetryIntervalSec 65 `
```

Comment:
We might know when the rate limit will be reset from response header's `x-ratelimit-reset`.
If the value is around 65, it will be better. Or how about using just 60 seconds because rate limit seems to be reset evert 60 seconds?   
https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28#checking-the-status-of-your-rate-limit

Reply:
`x-ratelimit-reset` header was not included in the response.
Maybe because it is not an API.

I'm guessing that the limit is 60 seconds, but I set it a little more at 65 seconds because it might be denied if it is just right.

Reply:
Thank you for checking it. It sounds nice to me!